apps/docs/content/dotnet/how-to/build-pipeline.mdx

```diff
@@ -356,7 +356,7 @@ deployFiles: ./path/~/to/
 
 Add a `.deployignore` file to the root of your project and specify which files and folders Zerops should ignore during deploy. The syntax is the same as of a <a href="https://git-scm.com/docs/gitignore#_pattern_format">`.gitignore`</a> file.
 
-Example to ignore `src/file.txt` file in deploy:
+Example to ignore `/src/file.txt` file in deploy:
```

Comment:
It still needs more clarification, why is there the leading `/` and what happens if you don't include it